server/config.ts

```diff
@@ -116,5 +116,8 @@ export default {
     // someToggleEnabled: Boolean(get('SOME_TOGGLE_ENABLED', false)),
     workAndInterestsTabEnabled: Boolean(get('WORK_AND_INTERESTS_TAB_ENABLED', false)),
     otherQualificationsEnabled: Boolean(get('OTHER_QUALIFICATIONS_ENABLED', false)),
+    frontendComponentsApiToggleEnabled: Boolean(
```

Comment:
Defo set the default to false here, for the same reasons as above (but also because we are not deploying to preprod and you dont have an override set in values-preprod) (we 100% dont want it in preprod yet)

Reply:
Fixed